ui/src/core/helpers/axios.helper.ts

```diff
@@ -5,6 +5,7 @@ import { env } from '@constants';
 
 const _axios = axios.create({
   baseURL: env?.BACKEND_BASE_URL?.toString(),
+  reuseBrowserDomain: true,
```

Comment:
What if we want to send request to a third party api .

Reply:
:thinking: do we? or would the backend do that? but good point i'll remove this for now as i'm not sure if this would screw with ory.